src/components/OverviewPageDeviceDetails.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
   project: string;
 }
 
-const InstanceOverviewDeviceDetail: FC<Props> = ({ device, project }) => {
+const OverviewPageDeviceDetails: FC<Props> = ({ device, project }) => {
```

Comment:
```suggestion
const DeviceDetails: FC<Props> = ({ device, project }) => {
```
A bit more concise and just as precise?